common/rpc/interceptor/telemetry.go

```diff
@@ -215,10 +215,14 @@ func (ti *TelemetryInterceptor) RecordLatencyMetrics(ctx context.Context, startT
 		userLatencyDuration = time.Duration(val)
 		metrics.ServiceLatencyUserLatency.With(metricsHandler).Record(userLatencyDuration)
 	}
+	handoverRetryLatency := time.Duration(0)
+	if val, ok := metrics.ContextCounterGet(ctx, metrics.NamespaceHandoverRetryLatency.Name()); ok {
+		handoverRetryLatency = time.Duration(val)
```

Comment:
so we don't actually emit metric for this latency?

Reply:
Follow the command to exclude the backoff retry delay from the no user latency.

Reply:
We need to chat about this.